bandit/formatters/csv.py

```diff
@@ -24,10 +24,10 @@
 .. code-block:: none
 
     filename,test_name,test_id,issue_severity,issue_confidence,issue_text,
-    line_number,line_range
+    more_info,line_number,line_range
```

Comment:
I think more_info should be listed after the line_range.  That should help those that may be parsing the CSV with the introduction of a new column.

Reply:
done